reports/models.py

```diff
@@ -0,0 +1,20 @@
+from sre_parse import CATEGORIES
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class Report(models.Model):
+    """
+    Name and file path of generated reports.
+    """
+
+    name = models.CharField(max_length=20)
+    directory = models.CharField(max_length=250)
+    user_id = models.ForeignKey(User)
+    category = models.CharField(max_length=50)
```

Comment:
Are categories user defined or system defined?

Reply:
Reports are not defined by users so should be system defined.